sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -663,14 +662,10 @@ def split(self, desired_bundle_size, start_position=None, stop_position=None):
         self._setup_temporary_dataset(bq)
         self.table_reference = self._execute_query(bq)
 
-      schema, metadata_list = self._export_files(bq)
+      unused_schema, metadata_list = self._export_files(bq)
```

Comment:
Does it mean that coder is no longer needed for `_CustomBigQuerySource`?

Reply:
You're correct!

Reply:
This is great! That would mean that we can safely remove `_JsonToDictCoder`. Do you think you can do this as a part of this PR?

Reply:
We may do that, but if we end up keeping a backwards compatibility flag, we'll need to keep the coder as-is.

Reply:
I see. Then let's wait for the decision.